README.md

```diff
@@ -541,7 +541,8 @@ To run:
 
 ### Setup Portal from Scratch
 
-Setup process requires 3 playbooks:
+Setup process requires 3 playbooks (or 2 playbooks if you enable portal deploy
+during portal setup, see playbook `portal-setup-following` below):
```

Comment:
I would revert this.

New portals can't run the deploy as part of setup following in their current state because of the consensus needing to sync.

Reply:
done